scripts/caravan/event-handlers/action.lua

```diff
@@ -143,7 +190,18 @@ local function on_confirmed(event)
         action.item_count = value
     end
 
-    if player.gui.screen[prefix .. "_frame"] then 
+    -- keep slider in sync after setting the action value 
+    do
```

Comment:
What's the purpose of wrapping in a `do` here?

Reply:
Removed do